channels/stable-4.2.yaml

```diff
@@ -1,6 +1,11 @@
 name: stable-4.2
 versions:
+- 4.1.20
+- 4.1.21
+- 4.1.22
+- 4.1.23
```

Comment:
This can't go into stable without also going into fast.

Reply:
/hold

Reply:
ack, thanks